scalafix-tests/input/src/main/scala/test/DisableSyntax.scala

```diff
@@ -7,19 +7,20 @@ DisableSyntax {
     return
     throw
   ]
-  tabs = true
-  semicolons = true
-  xml = true
+  noTabs = true
+  noSemicolons = true
+  noXml = true
 }
 */
 package test
 
 case object DisableSyntax {
-  var a = 1                 // assert: DisableSyntax.var
-  null                      // assert: DisableSyntax.null
-  def foo: Unit = return    // assert: DisableSyntax.return
-  throw new Exception("ok") // assert: DisableSyntax.throw
+  var a = 1                 // assert: DisableSyntax.keywords.var
```

Comment:
I think I preferred  `DisableSyntax.var`, what was the motivation here?

Reply:
to be consistent with the configuration. I think this way it's easier to see where it came from.

Reply:
Fair enough. I've wondered if `// scalafix:ok var` should be enough to disable `DisableSyntax.keywords.var`, since the latter is quite verbose. However, I just checked and `// scalafix:ok var` actually is enough to disable this message, which is good.

Reply:
I have wondered whether the full category id should be printed to the console since it does add a lot of bloat

```scala
scala/test/Disable.scala:17: error: [Disable.asInstanceOf] asInstanceOf is disabled
  val w = List(1, 2, 3).asInstanceOf[Seq[String]] // assert: Disable.asInstanceOf
```

but that's unrelated to this PR :)

Reply:
This is why we should add the error message to the testkit :). You can still use assert:foo but you can also output the full message to tune the message UX.

Reply:
you can already see the error messages in testkit ;) asserting against the full message contents is overtesting IMO, testkit should make it as easy as possible to add tests, copy-pasting the message or tweaking the caret adds an overhead which might demotivate people to add more tests.

Reply:
It would be in addition to what we already have.